include/curl/multi.h

```diff
@@ -464,6 +464,20 @@ typedef int (*curl_push_callback)(CURL *parent,
                                   struct curl_pushheaders *headers,
                                   void *userp);
 
+/*
+ * Name:    curl_multi_pollfds()
```

Comment:
Since this function populates `curl_waitfd` structs, should it perhaps rather be called `curl_multi_waitfds` ? I mean, the data it returns is not carved out to be used by `poll()` only but can be used by "anything".

Reply:
Yes, I agree that the `curl_multi_waitfds` name indeed better represents what this function does.